packages/react/src/themes/teams/components/Dropdown/dropdownItemStyles.ts

```diff
@@ -59,6 +59,13 @@ const dropdownItemStyles: ComponentSlotStylesInput<DropdownItemProps, DropdownVa
     fontSize: v.listItemContentFontSize,
     color: v.listItemContentColor,
   }),
+  checkableIndicator: ({ variables: v }) => ({
+    position: 'relative',
+    left: pxToRem(3),
```

Comment:
The padding on the `DropdownItem` is `11px`, but the icon should be `8px` from the right border, that's why we need styles